curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md

```diff
@@ -32,11 +32,11 @@ Fulfill the user stories below and get all the tests to pass to complete the lab
 
 9.  The `.orbit` element should have a width and height of `200px`.
 
-10.  The `.orbit` element should be positioned using `absolute` positioning. Its bottom right corner should be at the center of the `space` `div` using a `transform` property that shifts it by `-50%` on both the vertical and horizontal axes.
+10.  The `.orbit` element should be positioned using `absolute` positioning. Its bottom right corner should be at the center of the `space` element using a `transform` property that shifts it by `-50%` on both the vertical and horizontal axes.
```

Comment:
```suggestion
10.  The `.orbit` element should be positioned using `absolute` positioning. Its bottom right corner should be at the center of the `.space` element using a `transform` property that shifts it by `-50%` on both the vertical and horizontal axes.
```

class names always need the period, please don't forget to do this change in all the places